web/react/components/post_list.jsx

```diff
@@ -416,9 +417,9 @@ module.exports = React.createClass({
                         <div className='channel-intro'>
                             <h4 className='channel-intro__title'>Beginning of {uiName}</h4>
                             <p className='channel-intro__content'>
-                                Welcome to {uiName}!
+                                Welcome to <strong>{uiName}</strong>!
                                 <br/><br/>
-                                {'This is the first channel ' + strings.Team + 'mates see when they'}
+                                This is the first channel {strings.Team} mates see when they
```

Comment:
Is the added space between {strings.Team} and mates intentional?  Believe it should be teammates with no space.